org.kde.kwave.json

```diff
@@ -0,0 +1,112 @@
+{
+    "id": "org.kde.kwave",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-21.08",
+    "sdk": "org.kde.Sdk",
+    "command": "kwave",
+    "rename-icon": "kwave",
+    "finish-args": [
+        "--device=dri",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland"
+    ],
+
+    "modules": [
+        "shared-modules/libmad/libmad.json",
+        {
+            "name": "audiofile",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "http://audiofile.68k.org/audiofile-0.3.6.tar.gz",
+                    "sha256": "cdc60df19ab08bfe55344395739bb08f50fc15c92da3962fac334d3bff116965"
+                },
+                {
+                    "type": "patch",
+                    "path": "audiofile-gcc6.patch"
+                }
+            ]
+        },
+        {
+            "name": "id3lib",
+            "buildsystem": "simple",
+            "build-commands": [
+                "libtoolize -fc",
+                "aclocal",
+                "autoconf",
+                "automake --add-missing --copy",
+                "./configure --prefix=/app",
+                "make",
+                "make install"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://downloads.sourceforge.net/sourceforge/id3lib/id3lib-3.8.3.tar.gz",
+                    "dest-filename": "id3lib.tar.gz",
+                    "sha256": "2749cc3c0cd7280b299518b1ddf5a5bcfe2d1100614519b68702230e26c7d079"
+                },
+                {
+                    "type": "patch",
+                    "path": "id3lib_patches/05-create-m4-directory.patch"
```

Comment:
you can collapse all the patches into a single entry with `paths` and an array of all the patch files in order

Reply:
Thanks for the suggestion!